### DISCUSSION TITLE
Detect if currently streaming bluetooth audio?

### DISCUSSION THREAD

**Discussion by clarkbremer** [pschatzmann/ESP32-A2DP#167] [Q&A] [answered]
I'm using A2DP Sink, and wondering if there's a way to tell if we are currently streaming audio or not?

**Comment by pschatzmann** [comment, marked as answer]
get_audio_state ()
or you can register a corrsponding event handler..
https://pschatzmann.github.io/ESP32-A2DP/html/class_bluetooth_a2_d_p_sink.html

**Comment by clarkbremer** [comment]
Perfect, thanks.  For anyone else who stumbles in here, the possible results are:

ESP_A2D_AUDIO_STATE_STARTED
ESP_A2D_AUDIO_STATE_STOPPED
ESP_A2D_AUDIO_STATE_REMOTE_SUSPEND
